sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/SecHubConfigurationModelValidator.java

```diff
@@ -284,41 +285,108 @@ private void handleHTTPHeaders(InternalValidationContext context, SecHubWebScanC
             return;
         }
 
+        String targetUrlLowerCased = addTrailingSlashToUrlWhenMissingAndLowerCase(webScan.getUrl().toString());
+
+        // A simple map to keep track of headers an the urls they should be sent to
+        // With this map we validate the header configurations and inform the user about
+        // invalid combinations
+        Map<String, Boolean> occurencesMap = new HashMap<>();
+
         List<HTTPHeaderConfiguration> httpHeaders = optHttpHeaders.get();
         for (HTTPHeaderConfiguration httpHeader : httpHeaders) {
-            if (httpHeader.getName() == null || httpHeader.getName().isEmpty()) {
-                context.result.addError(WEB_SCAN_NO_HEADER_NAME_DEFINED);
-                return;
-            }
-            if (httpHeader.getValue() == null || httpHeader.getValue().isEmpty()) {
-                context.result.addError(WEB_SCAN_NO_HEADER_VALUE_DEFINED);
+            if (!assertHeaderNameAndHeaderValueAreSet(context, httpHeader)) {
```

Comment:
```suggestion
            if (!validateHeaderNameAndHeaderValueAreSet(context, httpHeader)) {
```
When I see an assert method, I always think about "it fails, throws an exception when a problem is there and it has not return value" .
So please rename ...